Functions/Quantification/xASL_quant_HadamardDecoding.m

```diff
@@ -46,8 +46,8 @@
 
 % Specify the TimeEncodedMatrix
 if isfield(xQ,'TimeEncodedMatrix') && ~isempty(xQ.TimeEncodedMatrix)
-	if (size(xQ.TimeEncodedMatrix,1)+1) ~= size(xQ.TimeEncodedMatrix,2)
-		error('TimeEncodedMatrix must be square (N-1)xN ');
+	if (size(xQ.TimeEncodedMatrix,1)) ~= size(xQ.TimeEncodedMatrix,2)
+		error('TimeEncodedMatrix must be square NxN ');
```

Comment:
You mean square MxN right?

Reply:
Are you sure you don't want to google the definition of a square? :)

The matrix needs to be indeed a square (NxN), not a rectangle (MxN).

Reply:
OK. So you can resolve...